### ISSUE TITLE
NTR: Chitotriosidase high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8273]
New term request Chitotriosidase high in blood (elthat_110211032106):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000611
- label: Elevated circulating chitotriosidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:31906975
- def: Increased activity or concentration of the enzyme chitotriosidase. Serum chitotriosidase is a biomarker that has shown high specificity and sensitivity in patients with sarcoidosis.
- comment: Significantly higher chitotriosidase activity can be observed in sarcoidosis patients than in healthy controls.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8273
- synonyms:
